src/content/plugins/split-chunks-plugin.md

```diff
@@ -180,7 +180,7 @@ A module can belong to multiple cache groups. The optimization will prefer the c
 
 `boolean`
 
-Reuse existing chunks instead of creating a new one when modules match exactly.
+If one of your chunks contains only modules that are already going to be split out, it will be reused instead of a new one being generated. This can impact the resulting file name of the chunk.
```

Comment:
I would rephrase this a little bit, is a little confusing.

Maybe:

_Reuses current chunk if it contains modules already splitted from main bundle instead of generating a new one. This can impact the resulting file name of the chunk._